src/main/java/collab/Employee.java

```diff
@@ -11,11 +12,88 @@
     private String lastName;
     private String firstName;
     private int realEmployeeNumber;
-    private int middlePhoneNumber;
-    private int lastPhoneNumber;
-    private int birthYearOnly;
-    private int birthMonthOnly;
-    private int birthDayOnly;
+    private String middlePhoneNumber;
+    private String lastPhoneNumber;
+    private String birthYearOnly;
+    private String birthMonthOnly;
+    private String birthDayOnly;
+
+    public Employee(List<String> commandArguments) {
+        employeeNumber = commandArguments.get(0);
+        name = commandArguments.get(1);
+        careerLevel = commandArguments.get(2);
+        phoneNumber = commandArguments.get(3);
+        birthday = commandArguments.get(4);
+        certi = commandArguments.get(5);
+        dataProcessing();
+    }
+
+    private void dataProcessing(){
+        processEmployeeNumber();
+        processName();
+        processPhoneNumber();
+        processBirthday();
+    }
+
+    private int convertRealEmployeeYear(int employeeNumberYearInt){
+        int realEmployeeNumberYearInt;
+        int currentYear = 2022;
+        int currentCentury = 2000;
+        int lastCentury = 1900;
+
+        if (employeeNumberYearInt <= (currentYear - currentCentury) && employeeNumberYearInt >= 0){
+            realEmployeeNumberYearInt = currentCentury + employeeNumberYearInt;
+        } else {
+            realEmployeeNumberYearInt = lastCentury + employeeNumberYearInt;
+        }
+        return realEmployeeNumberYearInt;
```

Comment:
else문을 이렇게 제거하면 어떨까요?

```suggestion
        if (employeeNumberYearInt <= (currentYear - currentCentury) && employeeNumberYearInt >= 0){
            return currentCentury + employeeNumberYearInt;
        }
        return lastCentury + employeeNumberYearInt;
```

Reply:
좋은 의견 감사드립니다! 추후 리팩토링시 반영하도록 하겠습니다!